packages/player/src/components/cue-form.tsx

```diff
@@ -0,0 +1,151 @@
+import {useSelector} from '@xstate/react'
+import * as React from 'react'
+import {useVideo} from '../context/video-context'
+import {
+  selectCurrentTime,
+  selectIsSubmittingCueNote,
+  selectCueNoteVisibility,
+  selectViewer,
+} from '../selectors'
+
+export const CueForm: React.FC = () => {
```

Comment:
this is the form ui for authoring notes

<img src="https://user-images.githubusercontent.com/25487857/149931579-825c02a9-12c7-4ce1-a41d-28aefaa0ad08.png" width="500">